identity/handler_test.go

```diff
@@ -673,6 +675,21 @@ func TestHandler(t *testing.T) {
 		}
 	})
 
+	t.Run("case=should fail to update identity if input json is empty or json file does not exist", func(t *testing.T) {
```

Comment:
This test case does not explicitly handle the case of non existing json file input. Because even if we have nil file pointer, the **send()** method is expecting **interface type** as 3rd parameter. And converting a file pointer to interface type would be challenging based on my current understanding.